### ISSUE TITLE
Layer

### ISSUE THREAD

**Issue by LouisAmon** [nuage-studio/lambda-packaging#4]
By default, `python -m lambda-layer` should indeed package a Lambda and/or package dependencies separately via docker (cf. #3)

In a CI though, one may want to package only the Layer (because requirements.txt checksum changed) or only the Lambda, or both.

In order to do that, I'm thinking a `python -m lambda-packaging --layer-only` argument may be added

An alternative way would be to create a `python -m layer-packaging` command which takes it's own arguments (e.g. path to requirements) but then we'd have to use inheritance mechanisms to ensure DRYness

It's up to debate but either way, it should be possible to use any of the above 3 scenarii is a pure-python and efficient way.

**Comment by cloudspeak** [comment]
I could be wrong, but I think this case is already covered by the existing options in #2.  If we give a value for `layer-output` and `requirements`, but `output` is not set, then only the layer will be generated.

Having said that, I've realised that this might be inconvenient if you have your `.lambda-packagerc` file set up the way you want it but need to run it with two different sets of options.  So I guess what we're saying is, the `layer-only` option effectively overrides `output` to be `None`.  Does that sound about right?

**Comment by LouisAmon** [comment]
Yup, that sounds about right : it's technically available but not properly exposed through an explicit interface

`output` is a bit deceptive when you want a Layer because you may expect this to be the path to your packaged layer

Maybe it's best to name it `lambda-output`, or maybe support both... IDK it's a bit verbose too

What do you think ?
